### PR TITLE
creates user folder and renames user components

### PR DESCRIPTION
**Pull request by lucas-wills**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const Root = ({store}) => {
 					<Route path = '/' component={App}>
 						<IndexRoute component={Home} />
 						<Route path = '/newUser' component={NewUser} />
```
Comment (janie-t) [inline src/index.js:33]: Perhaps you should change the newUser component also. 
We should make a ticket to change the routes to match components.

File: src/index.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ const App = require('./components/app')
 const NewUser = require('./components/containers/newUserPage')
 const NightOut = require('./components/containers/nightOutPage')
 const NewNightOut = require('./components/containers/newNightOut')
-const UserProfile = require('./components/userProfile')
-const UsersList = require('./components/usersList')
+const UsersProfile = require('./components/users/profile')
+const UsersList = require('./components/users/list')
```
Comment (janie-t) [inline src/index.js]: lets change this to index instead of list

**Review by janie-t — CHANGES_REQUESTED** [review]
Changes look good!  I have left some comments. When I start this app I get this error:
```
Uncaught Error: Cannot find module "./newLink"(…)
```

Can you fix it and let me know when its done, thanks!
:tropical_fish: :cactus: :poodle:

File: src/index.js (hunk 2 above)
Comment (lucas-wills) [reply src/index.js]: Thanks, I have now done this!

File: src/index.js (hunk 1 above)
Comment (lucas-wills) [reply src/index.js:33]: I have changed the newUser component also!

**Review by janie-t — APPROVED** [review]

**Comment by janie-t** [comment]
Tu meke it all works well done
:fireworks: 
:laughing:

**Pull request merged**